arthur_bench/scoring/hedging_language.py

```diff
@@ -28,10 +39,6 @@ def name() -> str:
     def requires_reference() -> bool:
         return False
 
-    def __init__(self):
-        with suppress_warnings("transformers"):
-            self.scorer = BERTScorer(lang="en", model_type=DEFAULT_MODEL)
-
     def to_dict(self, warn=False):
         return {"model_type": self.scorer.model_type}
```

Comment:
This method determines the config that is saved when a test suite is created using this scorer. That config will be used to reinitialize the scorer for future runs, so the hedging language used should additionally be added to the config here

Reply:
Oh got it, thanks. Done!